resources/views/layouts/partials/header/autocomplete/no-results.blade.php

```diff
@@ -0,0 +1,19 @@
+{{-- When there are no results in the search autocomplete this will be shown. --}}
+<div class="font-bold text-neutral text-lg">@lang('There are no results found for') <span class="text-primary">"@{{ value }}"</span></div>
```

Comment:
`No results found for ...`

Also please try to keep the span with the search term inside of the translation because not every language will be able to support this formatting.